portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,77 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+ *{
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+    } 
```

Comment:
Indent these lines back a few spaces until the closing brace `}` is all the way to the left.